rpm/usb-moded.spec

```diff
@@ -1,5 +1,5 @@
 Name:     usb-moded
-Version:  0.86.0+mer67
+Version:  0.86.0+mer68
```

Comment:
Could update the mer-core url below while at it

Reply:
Added "cleanup spec" commit